src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,38 @@
+package Duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    void printWelcome() {
+        System.out.println("Hello I'm Duke\nWhat can I do for you?");
+    }
+
+    void acceptInput() {
+        Scanner SC = new Scanner(System.in);

```

Comment:
Variable names must be in camelCase, fully uppercase names are reserved for constants.